### PR TITLE
LRAC-13920 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://issues.liferay.com/browse/LRAC-13920

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/onboarding-modal/ConnectDXP.tsx [hunk 1]
```diff
@@ -368,26 +368,15 @@ const Footer: FC<IFooterProps & IConnectDXPProps> = ({
...
-					>
-						{Liferay.Language.get('done')}
-					</ClayLink>
-				)}
+				<ClayLink
+					button
+					className='button-root'
+					displayType='primary'
+					href={getNavHref()}
+					onClick={() => onClose()}
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/onboarding-modal/ConnectDXP.tsx:376]:
```suggestion
					onClick={onboarding ? onNext : onClose}
```

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/onboarding-modal/ConnectDXP.tsx [hunk 2]
```diff
@@ -368,26 +368,15 @@ const Footer: FC<IFooterProps & IConnectDXPProps> = ({
...
+				>
+					{Liferay.Language.get('done')}
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/onboarding-modal/ConnectDXP.tsx:378]:
```suggestion
					{onboarding ? Liferay.Language.get('next') : Liferay.Language.get('done')}
```

**Review by interaminense — CHANGES_REQUESTED** [review]
Thanks @marcosapmf, just few comments in your PR.

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @marcosapmf

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**